### PR TITLE
chore: fix cases with desc and show

### LINKED ISSUE #7254: describle table show return results in logic test
**Summary**
https://github.com/datafuselabs/databend/blob/main/tests/logictest/suites/base/05_ddl/05_0000_ddl_create_tables#L163-L170

### PR DESCRIPTION
**Pull request by ZeaLoVe**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Fix some statement with show and desc has results but use statement ok.

Fixes #7254

### CODE REVIEW

**Comment by xudong963** [comment]
@mergify update

File: tests/logictest/suites/base/05_ddl/05_0016_ddl_stage [hunk 1]
```diff
@@ -13,15 +13,18 @@ CREATE STAGE test_stage_internal file_format=(type=csv compression=AUTO record_d
...
+----
+test_stage_internal Internal StageParams { storage: Fs(StorageFsConfig { root: "_data" }) } CopyOptions { on_error: None, size_limit: 0 } FileFormatOptions { format: Csv, skip_header: 0, field_delimiter: "", record_delimiter: "NONE", compression: Auto } 0 'root'@'127.0.0.1' 
+
+statement query TTTTT
 SHOW STAGES;
 
 ----
-test_stage External NULL 'root'@'127.0.0.1'
-test_stage_internal Internal 0 'root'@'127.0.0.1'
+test_stage External NULL 'root'@'127.0.0.1'
```
Comment (xudong963) [inline tests/logictest/suites/base/05_ddl/05_0016_ddl_stage:26]: Is the white space expected in the end?

Comment (ZeaLoVe) [reply tests/logictest/suites/base/05_ddl/05_0016_ddl_stage:26]: > Is the white space expected in the end?

Yes, actually there are 5 columns, the last one is empty.

**Review by xudong963 — APPROVED** [review]
Btw, is there a bug in auto-generated script?

**Review by flaneur2020 — APPROVED** [review]

**Pull request merged**